public/app/plugins/panel/geomap/editor/layerEditor.tsx

```diff
@@ -1,7 +1,7 @@
 import { get as lodashGet, isEqual } from 'lodash';
 
 import { FrameGeometrySourceMode, getFrameMatchers, MapLayerOptions } from '@grafana/data';
-import { NestedPanelOptions, NestedValueAccess } from '@grafana/data/src/utils/OptionsUIBuilders';
+import { NestedOptions, NestedValueAccess } from '@grafana/data/src/utils/OptionsUIBuilders';
```

Comment:
^^^ another search and replace error?

Reply:
i'm sure - i only ran basic linting across all these, and scanned over them - didn't meticulously go one by one. Sorry, that was sloppy of me :/